From cfd33a0c06f894ad0d3940842fce633e1b10a7c7 Mon Sep 17 00:00:00 2001
From: Paul Barker <paul@betafive.co.uk>
Date: Mon, 21 Oct 2019 10:41:29 +0100
Subject: [PATCH 4/4] vc-graphics: Clarify license conditions

The license text matches the 'Broadcom-RPi' custom license used in
bcm2835-bootfiles.

Signed-off-by: Paul Barker <paul@betafive.co.uk>
---
 recipes-graphics/vc-graphics/vc-graphics.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/recipes-graphics/vc-graphics/vc-graphics.inc b/recipes-graphics/vc-graphics/vc-graphics.inc
index 8b3f2bf..540e289 100644
--- a/recipes-graphics/vc-graphics/vc-graphics.inc
+++ b/recipes-graphics/vc-graphics/vc-graphics.inc
@@ -1,5 +1,5 @@
 DESCRIPTION = "Graphics libraries for BCM2835."
-LICENSE = "Proprietary"
+LICENSE = "Broadcom-RPi"
 
 LIC_FILES_CHKSUM = "file://LICENCE;md5=86e53f5f5909ee66900418028de11780"
 
-- 
2.23.0

